clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -783,6 +793,79 @@ trait GvcfTests { self: BaseIntegrationSpec =>
     }
   }
 
+  it should "automatically set regulatory designation to ResearchOnly for gvcfs" in {
+    val project = s"project$randomId"
+    val sample = s"sample$randomId"
+    val version = 3
+
+    val cloudPath = rootTestStorageDir.resolve(
+      s"gvcf/$project/$sample/v$version/$randomId.gvcf"
+    )
+
+    val key = TransferGvcfV1Key(Location.GCP, project, sample, version)
+    val metadata =
+      TransferGvcfV1Metadata(
+        gvcfPath = Some(cloudPath.toUri),
+        regulatoryDesignation = None
+      )
+
+    def query = {
+      for {
+        response <- runClient(ClioCommand.queryGvcfName, "--project", project)
+        results <- Unmarshal(response).to[Seq[TransferGvcfV1QueryOutput]]
+      } yield {
+        results should have length 1
+        results.head
+      }
+      Seq(1).mkString()
+    }
+
+    for {
+      _ <- runUpsertGvcf(key, metadata)
+      result <- query
+    } yield {
+      result.regulatoryDesignation should be(
+        Some(RegulatoryDesignation.ResearchOnly)
+      )
+    }
+  }
+
+  it should "respect user-set regulatory designation for gvcfs" in {
+    val project = s"project$randomId"
+    val sample = s"sample$randomId"
+    val version = 3
+
+    val cloudPath = rootTestStorageDir.resolve(
+      s"gvcf/$project/$sample/v$version/$randomId.gvcf"
+    )
+
+    val key = TransferGvcfV1Key(Location.GCP, project, sample, version)
+    val metadata =
+      TransferGvcfV1Metadata(
+        gvcfPath = Some(cloudPath.toUri),
+        regulatoryDesignation = Some(ClinicalDiagnostics)
```

Comment:
Nitpick: all the other usages have `RegulatoryDesignation.` in front of the enum value.